modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: REFERENCE
+
+[id="ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds_{context}"]
+= Integration between shared resources, Insights Operator, and OpenShift Builds
```

Comment:
```suggestion
= Integration between Shared Resources, Insights Operator, and {product-title} Builds
```

Reply:
Titles and "shared resources" use sentence case. Updated  "OpenShift Builds" to "{product-title} Builds." Thanks.